tools/markdown-magic/src/md-magic.config.cjs

```diff
@@ -254,6 +275,12 @@ function libraryPackageReadmeTransform(content, options, config) {
 	if (options.experimentalPackage === "TRUE" || packageScope === `@fluid-experimental`) {
 		sections.push(generateExperimentalPackageNotice());
 	}
+	if (options.internalPackage === "TRUE" || packageScope === `@fluid-internal`) {
+		sections.push(generateInternalPackageNotice());
+	}
+	if (options.privatePackage === "TRUE" || packageScope === `@fluid-private`) {
+		sections.push(generatePrivatePackageNotice());
+	}
```

Comment:
Not necessarily for this PR, but should these flags (and experimental) be mutually exclusive so a package can't end up with more than one of these notices?

Reply:
Yeah, probably. I'd initially dismissed that idea because the flags can be specified independently, but it would probably be best to assert if a user specifies more than one. Will update.

Reply:
I refactored the setup a bit. For the entire-README level pragma, there is a single scope argument, rather than 3 separate flags. They are now handled exclusively, and the specified argument always takes precedence over inheriting from the scope.

Reply:
I also replaced the individual transform pragmas for the different package scopes with a single `README_PACKAGE_SCOPE_NOTICE` pragma that does inherits from the package namespace when a scope kind is not explicitly specified. This will be nice if we ever rename packages between namespaces - the docs will auto-update.